validator/keymanager/types.go

```diff
@@ -85,7 +86,7 @@ func (k Kind) String() string {
 	case Derived:
 		return "derived"
 	case Local:
-		return "direct"
+		return "local"
```

Comment:
We need to support direct, as the majority of Prysm users still have this. The goal is that we support direct, local, and imported, as the same thing...Im not 100% sure how changing this line of code will affect existing validators. Need to think about this more

Reply:
I will revert and leave a comment regarding this.